02_configure_host.sh

```diff
@@ -123,6 +123,11 @@ for port in 6180 5050 6385 9999; do
     fi
 done
 
+# Add firewall rules to allow access to registry
```

Comment:
We could perhaps rebase this on your firewall PR since it's needed for testing on rhel hosts and looks close to landing?